src/main/kotlin/org/jetbrains/research/testspark/display/TestSparkDisplayFactory.kt

```diff
@@ -0,0 +1,55 @@
+package org.jetbrains.research.testspark.display
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.wm.ToolWindowManager
+import org.jetbrains.research.testspark.bundles.plugin.PluginMessagesBundle
+import org.jetbrains.research.testspark.core.data.Report
+import org.jetbrains.research.testspark.data.UIContext
+import org.jetbrains.research.testspark.display.coverage.CoverageVisualisationTabFactory
+import org.jetbrains.research.testspark.display.generatedTestsTab.GeneratedTestsTabFactory
+import javax.swing.JOptionPane
+
+class TestSparkDisplayFactory {
```

Comment:
This class is called `TestSparkDisplayFactory ` while it does not really follow Factory design pattern.

Reply:
Thank you, I agree. I changed Factories to Builders, I think, it's more suitable here